edk2toollib/windows/policy/firmware_policy_tool.py

```diff
@@ -0,0 +1,73 @@
+# @file
+# Generate the test firmware policy blobs for
+# the Firmware Policy Parsing Library Unit Test
+#
+# Copyright (c) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+from edk2toollib.windows.policy.firmware_policy import FirmwarePolicy
+import argparse
+
+
+def PrintPolicy(filename):
+    try:
+        with open(filename, "rb") as f:
+            policy = FirmwarePolicy(fs=f)
+            policy.Print()
+
+    except FileNotFoundError:
+        print('ERROR:  File not found: "{0}"'.format(filename))
+
+
+def CreatePolicyFromParameters(filename, manf, product, sn, nonce, oem1, oem2):
```

Comment:
Maybe print instead of create? We don't return the policy we create

Reply:
This is the CLI tool that Sean says I should move to extensions.
CreatePolicyFromParameters() initializes a Firmware Policy object with the parameters, opens a file "filename", and the Serialize(f) method writes it to file.  Print is here as a CLI debugging convenience.  My takeaway is that I should consider a better function name and add some documentation.  Noted, thanks!

Reply:
For now, I am going to remove this tool, and use this feedback when I PR it into Pytools extensions